src/Geno/C++/GUI/MainWindow.cpp

```diff
@@ -317,9 +317,9 @@ void MainWindow::ImGuiSettingsReadLineCB( ImGuiContext* /*pContext*/, ImGuiSetti
 	const char* pName = ( const char* )pEntry;
 	int         Bool;
 
-	if(      strcmp( pName, "Text Edit" ) == 0 ) { if( sscanf_s( pLine, "Active=%d", &Bool ) == 1 ) pSelf->pMenuBar->ShowTextEdit          = Bool; }
-	else if( strcmp( pName, "Workspace" ) == 0 ) { if( sscanf_s( pLine, "Active=%d", &Bool ) == 1 ) pSelf->pMenuBar->ShowWorkspaceOutliner = Bool; }
-	else if( strcmp( pName, "Output"    ) == 0 ) { if( sscanf_s( pLine, "Active=%d", &Bool ) == 1 ) pSelf->pMenuBar->ShowOutputWindow      = Bool; }
+	if(      strcmp( pName, "Text Edit" ) == 0 ) { if( sscanf( pLine, "Active=%d", &Bool ) == 1 ) pSelf->pMenuBar->ShowTextEdit          = Bool; }
+	else if( strcmp( pName, "Workspace" ) == 0 ) { if( sscanf( pLine, "Active=%d", &Bool ) == 1 ) pSelf->pMenuBar->ShowWorkspaceOutliner = Bool; }
+	else if( strcmp( pName, "Output"    ) == 0 ) { if( sscanf( pLine, "Active=%d", &Bool ) == 1 ) pSelf->pMenuBar->ShowOutputWindow      = Bool; }
```

Comment:
Here we can add defines again

Reply:
`sscanf` should exist on windows no? Its just the `_s` variant that's nonstandard

Reply:
Yes but it throws warnings about insecure calls, and since the signature is identical, we can just add a define instead of supplying `_CRT_SECURE_NO_WARNINGS`